doc/api/tls.markdown

```diff
@@ -395,34 +399,34 @@ Construct a new TLSSocket object from existing TCP socket.
 
 `function (response) { }`
 
-This event will be emitted if `requestOCSP` option was set. `response` is a
-buffer object, containing server's OCSP response.
+This event will be emitted if the `requestOCSP` option was set. `response` is a
+`Buffer` containing the server's OCSP response.
 
 Traditionally, the `response` is a signed object from the server's CA that
 contains information about server's certificate revocation status.
 
 ### Event: 'secureConnect'
 
-This event is emitted after a new connection has been successfully handshaked.
-The listener will be called no matter if the server's certificate was
-authorized or not. It is up to the user to test `tlsSocket.authorized`
-to see if the server certificate was signed by one of the specified CAs.
-If `tlsSocket.authorized === false` then the error can be found in
-`tlsSocket.authorizationError`. Also, if ALPN or NPN was used you can
-check `tlsSocket.alpnProtocol` or `tlsSocket.npnProtocol` for the
+This event is emitted after the handshaking process for a new connection has 
+successfully completed. The listener will be called whether the server's 
```

Comment:
How about "The listener will be called regardless of whether or not the server's certificate has been authorized"?